### ISSUE TITLE
Ad blocker issue

### ISSUE THREAD

**Issue by DianeLeo99** [photopea/photopea#6040]
Your Operating System: macOS Big Sur Version 11.7.10
Your Web Browser: Chrome
The problem happens in the Incognito Mode, too: yes, and i've tried Safari and Firefox

**This must be done to see the problem**
1. Go to www.photopea.com
2. It won't upload an image. 
3. Adblockker detected in the top right when I don't have any ad blocker. 
4. I've turned off all extensions.
5. I"m a digital marketer - I don't use an ad blocker because i need to see ads/
![Screen Shot 2023-09-14 at 8 38 59 pm](https://github.com/photopea/photopea/assets/144908196/0f5ecaad-44dc-4fed-a2bb-faec37a0f3dc)
![Screen Shot 2023-09-14 at 8 38 53 pm](https://github.com/photopea/photopea/assets/144908196/418f6f36-5a68-476d-a74f-c4187c51498b)

6. ....

**Comment by photopea** [comment]
Hi Diane, you should be able to use Photopea even with the Ad Blocker! 

How do you open a file? Do you use File - Open? What happens next? Is there any message? Did you try to open several different files, or just one file?

**Comment by camoody1** [comment]
@photopea As a side note to this issue... I was testing the Ad Block issue earlier and noticed that even when I white listed Photopea on my Ad Blocking extension, the sidebar still said that I had an Ad Blocker running and I only saw the games that you've developed. When I actually disabled the extension completely, though, then I started getting the full advertisement sidebar. So, some of these people may be whitelisting Photopea, but unless they totally disable it, they'll never see the real advertisements that you want them to.

I wonder why whitelisting doesn't work correctly for your page. That hurts you even when people TRY to do the right thing.
